### ISSUE TITLE
E712: False negative when multiple comparison in same line

### ISSUE THREAD

**Issue by friyaz** [PyCQA/pycodestyle#986]
Sample code:
```python
var = 1 if cond == True else -1 if cond == False else cond

if cond == False or cond == True:
    print(var)
```
When using pycodestyle 2.7.0
```
$ pycodestyle  mytest.py 
mytest.py:1:17: E712 comparison to True should be 'if cond is True:' or 'if cond:'
mytest.py:3:9: E712 comparison to False should be 'if cond is False:' or 'if not cond:'
```
When multiple equality comparisons are present in the same line, only first occurrence is detected.

**Comment by asottile** [comment]
try out #987 ?

**Comment by friyaz** [comment]
Thanks for the quick replay and PR @asottile. It is able to detect all occurrences.
